core/foundation/inc/TError.h

```diff
@@ -39,15 +39,14 @@
 
 class TVirtualMutex;
 
-constexpr Int_t kUnset    =  -1;
-constexpr Int_t kPrint    =   0;
-constexpr Int_t kInfo     =   1000;
-constexpr Int_t kWarning  =   2000;
-constexpr Int_t kError    =   3000;
-constexpr Int_t kBreak    =   4000;
-constexpr Int_t kSysError =   5000;
-constexpr Int_t kFatal    =   6000;
-
+R__EXTERN const Int_t kUnset;
```

Comment:
We can revert this commit: 3184f755abbb77e4ceff5b476dcf94c519dea25d Do we need to make the constants out-of-line? That'd break performance.

Reply:
Let me try with your suggestion, I will report the results from `strace`

Reply:
Unfortunately that is not enough. Here are the numbers:

Patch of this PR:
```
$: strace -z -f -o log.txt -e trace=openat python3 -c 'import ROOT;print(ROOT.kError);'
$: 3000
$: grep openat log.txt | wc -l
$: 323
```

Revert of https://github.com/root-project/root/commit/3184f755abbb77e4ceff5b476dcf94c519dea25d
```
$: strace -z -f -o log.txt -e trace=openat python3 -c 'import ROOT;print(ROOT.kError);'
$: 3000
$: grep openat log.txt | wc -l
$: 13297
```

Reply:
It really seems the discriminating factor is whether `kError` and other `k*` constants are present with symbols in `libCore.so` or not.